src/components/10-atoms/input-text/pw.ui.test.js

```diff
@@ -0,0 +1,25 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+
+describe('input-text', () => {
```

Comment:
In other input-text testcafe test, this describe starts with: `Input text` instead of `input-text`, although we do not have a common ground here anyways, I recommend to stick to what we have.